src/Raven.Server/Documents/Indexes/Index.cs

```diff
@@ -114,6 +114,9 @@ public abstract class Index : ITombstoneAware, IDisposable, ILowMemoryHandler
         private readonly ConcurrentDictionary<string, IndexProgress.CollectionStats> _inMemoryIndexProgress =
             new ConcurrentDictionary<string, IndexProgress.CollectionStats>();
 
+        private readonly ConcurrentDictionary<string, IndexProgress.CollectionStats> _inMemoryReferencesIndexProgress =
```

Comment:
Shouldn't we use `StringComparer.OridinalIgnoreCase` here (and in the other stats) ?

Reply:
Nvm, this comes from compiled.ReferencedCollections which has that compiler.

Reply:
Comparer*